tools/test_utils.py

```diff
@@ -95,6 +95,11 @@ def run_ngtf_pytests_from_artifacts(artifacts_dir):
     if not os.path.isdir(test_dir):
```

Comment:
Changes to this file shouldn't be needed.

Reply:
There are chdir(...) in scripts, so need to convert to absolute path early in the script-chain, in case user specifies relative path.

Reply:
The test scripts should convert to absolute path if they need to. Please avoid adding such special handling in the CI test runner.

Reply:
OK